package.json

```diff
@@ -4,28 +4,26 @@
   "main": "src/index.js",
   "license": "MPL-2.0",
   "scripts": {
-    "start": "cross-env NODE_ENV=production node ./server/index.js",
-    "dev": "cross-env NGROK=true node ./server/index.js",
-    "build": "webpack"
+    "dev": "webpack-dev-server --open --config webpack.dev.js",
+    "build": "webpack --config webpack.prod.js"
   },
   "dependencies": {
     "aframe-input-mapping-component": "https://github.com/fernandojsg/aframe-input-mapping-component",
     "aframe-teleport-controls": "https://github.com/netpro2k/aframe-teleport-controls#feature/teleport-origin",
-    "easyrtc": "^1.1.0",
-    "express": "^4.15.5",
-    "networked-aframe": "https://github.com/netpro2k/networked-aframe#feature/positional-audio",
-    "pleasejs": "^0.4.2",
-    "socket.io": "^2.0.3"
+    "naf-janus-adapter": "^0.1.1",
+    "networked-aframe": "https://github.com/netpro2k/networked-aframe#feature/register-adapter",
```

Comment:
Make sure this branch includes latest changes on master of main fork of networked-aframe as that includes positional audio changes.